docs/usage.md

```diff
@@ -110,6 +110,8 @@ to the pytest.
 * `--csv-change` - Allows changes in the OCS CSV. For example, usage of custom image,
    like MCG or RHCS. The format should be:
    <pattern_to_replace_from::pattern_to_replace_to>, while '::' is the delimiter
+* `--collect-logs-on-success-run` - Allows to have the option of gathering logs even,
```

Comment:
Can you move it right after line 104 to have this log collecting parameters together?

+ thinking about change the text to something like - Collect must gather logs at the end of the execution (also when no failure in the tests)